# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332787739 18000
#      Mon Mar 26 13:48:59 2012 -0500
# Node ID e2c25776392e3f51975cb39dc656dc478d4272f1
# Parent  f537eb145f63cab8a201227ea98ad20877641d8f
cmake: add metis library for linking to fix bug that appeared for static libraries not linking in the right order

diff -r f537eb145f63 -r e2c25776392e programs/CMakeLists.txt
--- a/programs/CMakeLists.txt	Mon Mar 26 13:48:12 2012 -0500
+++ b/programs/CMakeLists.txt	Mon Mar 26 13:48:59 2012 -0500
@@ -5,10 +5,10 @@ add_executable(mtest mtest.c io.c)
 add_executable(parmetis_prog parmetis.c io.c adaptgraph.c)
 set_target_properties(parmetis_prog PROPERTIES OUTPUT_NAME parmetis)
 add_executable(pometis pometis.c io.c)
 # Link to libparmetis.
 foreach(prog ptest mtest parmetis_prog pometis)
-  target_link_libraries(${prog} parmetis m)
+  target_link_libraries(${prog} parmetis metis m)
 endforeach(prog)
 
 install(TARGETS ptest mtest parmetis_prog pometis
   RUNTIME DESTINATION bin)
